docs/dataformats/observation_lists.rst

```diff
@@ -22,23 +22,48 @@ This has the advantage that everyone can work with whatever tool they like. Here
 * `csvkit <https://csvkit.readthedocs.org/en/latest/>`_
 * Your own script using e.g the `Python standard library CSV module <http://docs.python.org/2/library/csv.html>`_ or `pandas <http://pandas.pydata.org>`_
 
-A run list must have at least a column called ``Run``::
+A run list must have at least a column called ``OBS_ID``::
```

Comment:
Can you re-structure the whole page a bit?

Gammapy can work with observation tables in whatever format Astropy Table can read. So it could be CSV, but that doesn't have meta info, so at least for data distribution we'll use FITS (the current `runinfo.fits`).

So this general info about CSV tools should go to https://gammapy.readthedocs.org/en/latest/dataformats/file_formats.html#csv (if it's not all there already) and if you think it's useful, just keep a Sphinx `:ref:` link to that section from here.